changelog/3962.feature.rst

```diff
@@ -0,0 +1 @@
+``assert_truncate_level`` option added to be able to get the full diff without using ``-vv``.
```

Comment:
By using `confval` as I mentioned before, we can create a link to it automatically:

```suggestion
:confval:`assert_truncate_level` option added to be able to get the full diff without using ``-vv``.
```